src/GesturePanel/GesturePanel.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import Hammer from 'hammerjs';
+
+import './GesturePanel.css';
+
+class GesturePanel extends Component {
+
+    constructor(props) {
+        super();
+        this.myRef = React.createRef();
+        this.state = {
```

Comment:
should probably make a style variable instead of storing it in state incase a state variable is needed that is not stye

Reply:
Yeah, I saw that after. I think I pushed stuff in the Joystick panel that pulls straight from the props like how you mentioned. Might just do that